src/lib/components/Notifications/actions.js

```diff
@@ -1,11 +1,21 @@
+import get from 'lodash/get';
 import { isAPIError } from '@components/Error/Error';
+import { HTTP_STATUS_CODES_WITHOUT_NOTIFICATION } from '@api/base';
 
 export const ADD = 'notifications/ADD';
 export const REMOVE = 'notifications/REMOVE';
 export const CLEAR_ALL = 'notifications/CLEAR_ALL';
 
 export const sendErrorNotification = (error) => {
   return (dispatch) => {
+    const errorStatus = get(error, 'response.data.status');
+    if (
+      errorStatus !== undefined &&
+      HTTP_STATUS_CODES_WITHOUT_NOTIFICATION.includes(errorStatus)
+    ) {
+      dispatch(clearNotifications);
+      return;
+    }
```

Comment:
Minor: Just to confirm I understand correctly the purpose of this code, this is done to avoid showing a notification when redirecting to the 410 page? if that's the case I would have a look to the function `isAPIError` and modifiy the behaviour there rather than adding a custom behaviour here.

Reply:
I feel that the `isAPIError` function should just be a function to specify whether the response was an error or not as it's name suggests.
Adding the notification behaviour there might be making it more complex. What do you think?

Reply:
Alright, thanks for the insight here, I'll update the functions here and reuse them!